step-06/js/main.js

```diff
@@ -160,8 +160,10 @@ function signalingMessageCallback(message) {
 
   } else if (message.type === 'candidate') {
     peerConn.addIceCandidate(new RTCIceCandidate({
```

Comment:
This same logic is present in step-05/js/main.js -- could you add the change there as well?